alpine-consul/root/etc/services.d/consul-template/finish

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/execlineb -S1
+
+# only tell s6 to bring down the entire container, if it isn't already doing so
+# http://skarnet.org/software/s6/s6-supervise.html
```

Comment:
@smebberson not a big deal to me but my reasoning against would be that if
template  fails it means that configuration probably failed too and so
template in my mind is as essential as consul. In the end user defines
behavior so again not a big deal.
On Dec 16, 2015 4:46 PM, "Scott Mebberson" <notifications@github.com> wrote:

> In alpine-consul/root/etc/services.d/consul-template/finish
> <https://github.com/smebberson/docker-alpine/pull/13#discussion_r47856359>
> :
>
> > @@ -0,0 +1,8 @@
> > +#!/usr/bin/execlineb -S1
> > +
> > +# only tell s6 to bring down the entire container, if it isn't already doing so
> > +# http://skarnet.org/software/s6/s6-supervise.html
>
> @matthewvalimaki <https://github.com/matthewvalimaki>, I think it would
> be better if consul-template automatically restarted rather than bringing
> down the container (to do this just remove this file). consul-template is
> a secondary service to consul itself and I think it would be fine to have
> s6 automatically restart it.
>
> I think it would also go aways to resolving the consul-template runs
> before consul is ready issue (aside from the retry option I've just
> commented about).
>
> What do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/smebberson/docker-alpine/pull/13/files#r47856359>.
>